<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">margin:0;</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">background-color:white; border:none;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item alignment="Qt::AlignVCenter">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="styleSheet">
          <string notr="true">padding-left: 30px;padding-right:30px;</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="pushButton">
            <property name="styleSheet">
             <string notr="true">height:100;min-width:100;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>images/nda-logo.png</normaloff>images/nda-logo.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">font-size:18px;font-weight:500;</string>
            </property>
            <property name="text">
             <string>NDA CRIME DATABASE</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>FOR CARDETS</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QLineEdit" name="EmailInput">
            <property name="styleSheet">
             <string notr="true">color:grey;border:1px solid grey;min-height:40px!important;min-width:300px;border-radius: 20px;font-size:15px;</string>
            </property>
            <property name="text">
             <string>Email</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QLineEdit" name="PasswordInput">
            <property name="styleSheet">
             <string notr="true">color:grey;border:1px solid grey;min-height:40px!important;min-width:300px;border-radius: 20px;font-size:15px;</string>
            </property>
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="LoginButton">
            <property name="styleSheet">
             <string notr="true">background-color:#1AAA62;min-width: 300px;min-height:40px; border-radius:20px; border:none;color:white; font-size:15px</string>
            </property>
            <property name="text">
             <string>Login</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
